triagebot.toml

```diff
@@ -608,19 +608,18 @@ cc = ["@nnethercote"]
 [assign]
 warn_non_default_branch = true
 contributing_url = "https://rustc-dev-guide.rust-lang.org/getting-started.html"
-users_on_vacation = ["jyn514", "jackh726", "WaffleLapkin", "oli-obk"]
+users_on_vacation = ["jyn514", "WaffleLapkin", "oli-obk"]
 
 [assign.adhoc_groups]
 compiler-team = [
     "@cjgillot",
+    "@compiler-errors",
     "@petrochenkov",
     "@davidtwco",
     "@oli-obk",
     "@wesleywiser",
 ]
 compiler-team-contributors = [
```

Comment:
maybe these two should just be merged, i dont think its very useful to have them separate

Reply:
Yeah, there's basically no reason to assign a review specifically to a compiler team member. The team group in the organization can be pinged if a FCP needs started.

Reply:
I thought about it, and agree. Just didn't do it in this PR so its "trivial".